src/components/side-by-side-rendered-diff.jsx

```diff
@@ -24,11 +24,12 @@ export default class SideBySideRenderedDiff extends React.Component {
     if (!this.props) {
       return null;
     }
+    const onlyOneVersion = this.props.diff.from_version_id === this.props.diff.to_version_id;
 
     return (
       <div className="side-by-side-render">
         <iframe sandbox="allow-forms allow-scripts" ref={frame => this.frameA = frame} />
-        <iframe sandbox="allow-forms allow-scripts" ref={frame => this.frameB = frame} />
+        {(onlyOneVersion) ? null : <iframe sandbox="allow-forms allow-scripts" ref={frame => this.frameB = frame} />}
```

Comment:
I’m not really sure this is the best place to do this. I need to go implement the “highlighted rendered” diff now that the visual diff supports it.

That means implementing a diff that shows only single `<iframe>`, and that’s probably where we should send “side-by-side” diffs if we’re actually only looking at a single version.

Reply:
K, I'll fix this up when that PR is merged.